cargo-auditable/src/object_file.rs

```diff
@@ -24,6 +24,7 @@ pub fn create_metadata_file(
         b".dep-v0".to_vec(),
         SectionKind::ReadOnlyData,
     );
+    #[allow(clippy::single_match)]
```

Comment:
added this instead of apply the suggestion since this seems intentional

Reply:
I think it could actually be refactored into an `if let` without issues, so no reason to keep this match around as far as I can tell?